packages/fgtclb/academic-partners/Resources/Private/Backend/Partials/PageLayout/Doktype40.html

```diff
@@ -39,7 +38,12 @@
                                                 each="{categories}"
                                                 as="category"
                                             >
-                                                <li>{category.title}</li>
+                                            <li>
+                                                <f:if condition="{category.hidden}  == 1">
+                                                    <core:icon identifier="overlay-hidden" />
```

Comment:
An icon when hidden, but when not hidden no icon ? From ux point of view we should display a icon when not hidden to avoid questioning the difference from a editor point of view.